### PR TITLE
Elaborate on how to contribute

### PR DESCRIPTION
**Pull request by chickenbane**
Call out GitHub repo, location of github.io repos are by convention.  Expand the Ruby/Bundler section (Homebrew knowledge appears to be implied).  Clarify middle brew install step.

Thanks for starting this guide.  I wanted to contribute and while setting up I saw a few details that could be more clear.

### CODE REVIEW

**Comment by chickenbane** [comment]
I signed it!

File: contribute.md [hunk 1]
```diff
@@ -19,13 +19,18 @@ Pull requests should be made targeting the `master` branch. Every few weeks, the
 
 You can run the site locally on your computer while making changes.
 
+### Setup Ruby and Bundler
+
 Ensure that you have Ruby and [Bundler](http://bundler.io/) installed.
 
+    brew install ruby
```
Comment (JakeWharton) [inline contribute.md]: This assumes Mac OS which is why I didn't include it. I think it's safe to skip instructions for installing Ruby.

File: contribute.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Contributions to this website are welcome and appreciated!
 
 To contribute to this website, feel free to create pull requests for small fixes. For bigger contributions we recommend to start an issue on the [issue tracker](https://github.com/android/kotlin-guides/issues) first.
 
-Pull requests should be made targeting the `master` branch. Every few weeks, the [change log](changelog.html) will be updated and all changes in that time period will be released to the `gh-pages` branch.
+Pull requests should be made targeting the `master` branch from this [GitHub repo](https://github.com/android/kotlin-guides). Every few weeks, the [change log](changelog.html) will be updated and all changes in that time period will be released to the `gh-pages` branch.
```
Comment (JakeWharton) [inline contribute.md]: How about "of the" instead of "from this".

File: contribute.md [hunk 3]
```diff
@@ -19,13 +19,18 @@ Pull requests should be made targeting the `master` branch. Every few weeks, the
...
 
+    brew install ruby
+    gem install bundler
+
 ### One-time setup
 
     bundle install --path vendor/bundle
 
-_Note: If you're on Mac OS and this fails installing nokogiri, run `brew unlink xz`, install, and then `brew link xz`._
+_Note: If you're on Mac OS and this fails installing nokogiri, run `brew unlink xz`, `brew install xz`, and then `brew link xz`._
```
Comment (JakeWharton) [inline contribute.md]: This should be reverted. The install refers to running the `bundle install` command.

File: contribute.md (hunk 2 above)
Comment (chickenbane) [reply contribute.md]: ok

File: contribute.md (hunk 1 above)
Comment (chickenbane) [reply contribute.md]: removed

File: contribute.md (hunk 3 above)
Comment (chickenbane) [reply contribute.md]: reverted

**Pull request merged**